Board.h

```diff
@@ -32,11 +31,15 @@ class Board {
 
 	TopBoard* topB;
 
+	bool colored=true;
+
 	Board(int _player) : player(_player), gameZone(_player), gameFrame(gameZone), topB(new TopBoard(this)) { setBoard(); }; // ctr 
 
 	~Board() { delete topB; }; // dctr
 
-	
+	void setColored(bool _colored) {
+		colored = _colored;
```

Comment:
only private variables has abstraction functions like this one. colored is Public therefore you can access it directly. Maybe change it to private?

Reply:
changed